apps/system/js/rocketbar.js

```diff
@@ -459,7 +459,10 @@
      * @memberof Rocketbar.prototype
      */
     _handle_home: function() {
-      if (this.isActive()) {
+      if (this._activateCall) {
```

Comment:
Can we extract these 4 lines to a separate method? We have duplicated code in this file.